src/main/kotlin/no/nav/familie/ef/iverksett/infrastruktur/configuration/ApplicationConfig.kt

```diff
@@ -27,7 +27,7 @@ import java.time.Duration
 import java.time.temporal.ChronoUnit
 
 @SpringBootConfiguration
-@ConfigurationPropertiesScan("no.nav.familie.ef.iverksett.arena")
+@ConfigurationPropertiesScan("no.nav.familie.ef.iverksett.arena", "no.nav.familie.ef.iverksett.infrastruktur.configuration")
```

Comment:
For å unngå slitsom feilsøking i fremtiden synes jeg vi godt kan endre til denne :P 
`@ConfigurationPropertiesScan("no.nav.familie.ef.iverksett")`